rust/ittapi-sys/Cargo.toml

```diff
@@ -14,8 +14,11 @@ categories = ["development-tools", "external-ffi-bindings"]
 exclude = [
     "c-library/.git",
     "c-library/.github",
-    "c-library/.vscode",
```

Comment:
Can we leave this line in? I believe Cargo is getting confused here due to the `c-library` symlink and does not ignore this folder as it usually would. There may be some other trick to do this, but for those of us who use VS Code it is a safeguard against publishing private settings to crates.io.